### PR TITLE
Feed crowdstrike - added server url

### PR DESCRIPTION
**Pull request by abaumgarten**
## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/33962

## Description
Added a server URL as a not mandatory param.

## Minimum version of Demisto
- [ ] 5.0.0
- [x] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

Screen Shot 2021-02-21 at 10 17 37

### CODE REVIEW

**Comment by abaumgarten** [comment]
Labeling as `docs-approve' cause I took the RN from an identical issue that solved in another Content Pack in master.

File: Packs/FeedCrowdstrikeFalconIntel/Integrations/FeedCrowdstrikeFalconIntel/FeedCrowdstrikeFalconIntel.yml [hunk 1]
```diff
@@ -3,6 +3,11 @@ commonfields:
   id: Crowdstrike Falcon Intel Feed
   version: -1
 configuration:
+- defaultvalue: https://api.crowdstrike.com/
+  display: Server URL (e.g., https://example.net)
```
Comment (bakatzir) [inline Packs/FeedCrowdstrikeFalconIntel/Integrations/FeedCrowdstrikeFalconIntel/FeedCrowdstrikeFalconIntel.yml:7]:
```suggestion
  display: Cloud Base URL (e.g., https://api.crowdstrike.com)
```

File: Packs/FeedCrowdstrikeFalconIntel/ReleaseNotes/1_0_5.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Crowdstrike Falcon Intel Feed
+- You can now use custom URLs using the *Server URL* integration parameter.
```
Comment (bakatzir) [inline Packs/FeedCrowdstrikeFalconIntel/ReleaseNotes/1_0_5.md:4]:
```suggestion
- Added the **Cloud Base URL** integration parameter.
```

**Review by bakatzir — COMMENTED** [review]
let's conform to other integrations from the same vendor: https://github.com/demisto/content/pull/11412/files

**Review by bakatzir — COMMENTED** [review]
please add this param to the readme as well.

**Review by yuvalbenshalom — APPROVED** [review]

**Pull request merged**